spec/graph/post_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'explainer/parser'
+
+describe Post, type: :model do
+  let(:user) { create(:user) }
+  let(:posts) { FactoryGirl.create_list(:post, 5, author: user) }
+
+  it "should make N+1 queries for the authors of posts" do
+    user
```

Comment:
if you do `let!(:user) { ... }` above you won't need to do this.

Reply:
Ah fantastic thanks